scripts/ic-update/metadata.json

```diff
@@ -2,9 +2,9 @@
     "formatVersion": 1,
     "components": [
         {
-            "id": "h3ulcb-4x2g-ab-1.0-cluster",
+            "id": "rcar-multinode-zephyr-1.0-spider-cluster",
```

Comment:
Hi @iusyk, please remove the spider from the name.

Reply:
Done.

Reply:
why ? , this configuration works and has been tested, besides loo at the bord configuration:
 "id": "rcar-multinode-zephyr-1.0-spider-cluster"

Reply:
Hi @iusyk, you accidentally left a comment on the outdated topic. I'm mentioning this so that someone reading this PR would not be confused. ))